pytorch_blade/src/torch-mlir/lib/Conversion/TorchToMhlo/MhloLegalizeUtils.cpp

```diff
@@ -198,7 +198,7 @@ std::vector<Value> getDimSizesOfTensor(
   dimSizes.reserve(rank);
   auto loc = op->getLoc();
   for (auto d = 0; d < rank; ++d) {
-    auto d_size = currentKnowledge.sizes[d];
+    auto dSize = currentKnowledge.sizes[d];
```

Comment:
We do not use `dSize` in this for loop. Is it just unused code or a potential bug?